src/StructuredLogViewer.Avalonia/MainView.xaml.cs

```diff
@@ -0,0 +1,711 @@
+﻿using System;

```

Comment:
Unfortunately, this PR has lots of changes as I had to rename this file, so it can be reused in environments without windows (browser).